### ISSUE TITLE
Unable to compile on Windows

### ISSUE THREAD

**Issue by sooqua** [fireice-uk/xmr-stak-cpu#250]
I followed the instructions in readme, but I get this error:
`The application was unable to start correctly (0xc000007b). Click OK to close the application.`
I googled the error and it seems to have something to do with 32-bit DLLs being linked in the exe?
But the MSbuild says I am compiling 64-bit release version? And it compiles without any errors (30 warnings).
What am I doing wrong?

**Comment by sooqua** [comment]
I opened both my and your .exe files and I found that my .exe was having additional .dll's listed in it: libeay32.dll and ssleay32.dll. I found them here: C:\xmr-stak-dep\openssl\bin and I copied them to the .exe folder. Now it's running without any errors.
I wonder why that happens? My .exe is already bigger than yours, if I were to statically link these DLLs to my exe it would got even bigger, so I don't think that's what you did with your build.
